File: src/main/java/ru/isb/bot/utils/PathFileUtils.java
package ru.isb.bot.utils;

public class PathFileUtils {

    private static final String COPY_POSTFIX = "(копия)";

    public static String copyFileName(String fileName) {
        String [] prefixAndPostfix = fileName.split("\\.");
        return prefixAndPostfix.length == 2 ? prefixAndPostfix[0] + COPY_POSTFIX + "." + prefixAndPostfix[1] : fileName + COPY_POSTFIX;
    }

    public static String replaceSpace(String fileName) {
        return fileName.replace(" ", "_");
    }

}

File: src/main/java/ru/isb/bot/utils/PathFileUtils.kt
package ru.isb.bot.utils

class PathFileUtils {

    companion object {

        private val COPY_POSTFIX = "(копия)"

        fun copyFileName(fileName: String): String {
            val prefixAndPostfix = fileName.split('.')
            return if (prefixAndPostfix.size == 2) prefixAndPostfix[0] + COPY_POSTFIX + "." + prefixAndPostfix[1] else fileName + COPY_POSTFIX
        }

        fun replaceSpace(fileName: String): String =
            fileName.replace(" ", "_")

    }

}